deployer/src/main/java/org/apache/brooklyn/entity/cloudfoundry/webapp/PaasWebAppCloudFoundryDriver.java

```diff
@@ -65,7 +65,7 @@ private void initApplicationParameters() {
         if(!Strings.isBlank(getEntity().getConfig(CloudFoundryWebApp.APPLICATION_NAME))){
             applicationName = getEntity().getConfig(CloudFoundryWebApp.APPLICATION_NAME);
         } else {
-            applicationName = "cf-app" + Identifiers.makeRandomId(8);
+            applicationName = "cf-app-" + Identifiers.makeRandomId(8);
```

Comment:
Yes, CF application requires an unique id, if this id is not specify from the DAM, then the entity will generate and use this id